addon/initialize.js

```diff
@@ -12,11 +12,13 @@ function unwrap(input) {
 }
 
 export default function() {
-  ValidatorsMessages.reopen({
+  ValidatorsMessages.reopenClass({
```

Comment:
This hangs them off the constructor, which I don't think will work in this case.